ci/install-dependencies-windows.sh

```diff
@@ -19,7 +19,7 @@ PACKAGE="${PACKAGE:-false}"
 WIN_PLATFORM=x64-windows
 
 # install libraries
-vcpkg install netcdf-c gdal pcre2 fftw3[core,threads] clapack openblas --triplet ${WIN_PLATFORM}
+vcpkg install netcdf-c gdal[tools] pcre2 fftw3[core,threads] clapack openblas --triplet ${WIN_PLATFORM}
```

Comment:
Is only `gdal[tools]` needed here rather than `gdal[core,tools]`?

Reply:
`core` is always built, but it does no harm to use `gdal[core,tools]`.

Reply:
```suggestion
vcpkg install netcdf-c gdal[core,tools] pcre2 fftw3[core,threads] clapack openblas --triplet ${WIN_PLATFORM}
```